pkg/services/cloudmigration/api/api.go

```diff
@@ -450,16 +494,11 @@ func (cma *CloudMigrationAPI) GetSnapshotList(c *contextmodel.ReqContext) respon
 	}
 	q := cloudmigration.ListSnapshotsQuery{
 		SessionUID: uid,
-		Limit:      c.QueryInt("limit"),
-		Page:       c.QueryInt("page"),
-		Sort:       c.Query("sort"),
-		OrgID:      c.SignedInUser.OrgID,
-	}
-	if q.Limit == 0 {
-		q.Limit = 100
-	}
-	if q.Page < 1 {
-		q.Page = 1
+		Limit:      getQueryPageParams(c.QueryInt("limit"), 100),
```

Comment:
further cleanup requires cascading and xorm store changes, which seem too out of scope